exercises/calculator.cpp

```diff
@@ -5,8 +5,83 @@
   Insert first number: 4
   Insert second number: 2
 
-  SUM: 6
+  Sum: 6
   Difference: 2
   Multiplication: 8
   Division: 2
 */
+#include <string>
+#include <iostream>
+#include <limits>
+
+using namespace std;
+int main(){
+    string input = "";      //prima stringa, conterrà il primo operando o il comando di uscita
+    double output = 0;      //variabile in cui verrà conservato il risultato
+    double first = 0;       //primo operando (ottenuto da input dopo una conversione)
+    double second = 0;      //secondo operando
+    char operatore = ' ';   //carattere che determina l'operazione da effettuare
+    bool exit = false;      //booleano che ci segnala l'uscita dal programma
+    bool operation = true;  //booleano di supporto per determinare che un'operazione sia avvenuta (ovvero che sia stato inserito un operando valido)
```

Comment:
usa sempre l'inglese, anche nei commenti

cerca di dare nomi significativi alle variabili in modo da evitare di commentare il codice, idealmente il codice dovrebbe essere auto-esplicativo e comprensibile senza commenti